pkg/apply/resource_infos.go

```diff
@@ -70,7 +70,7 @@ func getIndexByKind(kind string) int {
 	return m[kind]
 }
 
-// Equals returns true if the GVK's have equal fields.
+// EqualsWithNormalize returns true if the GVK's have equal fields.
```

Comment:
Hmmm...doesn't match the comment does not match the function signature.

Reply:
Fixed